jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml

```diff
@@ -202,7 +202,7 @@ spec:
   tags:
   - from:
       kind: DockerImage
-      name: docker-registry.upshift.redhat.com/aicoe-notebooks/ocp4-anomaly-detection:latest
+      name: docker-registry.upshift.redhat.com/aicoe-notebooks/ocp4-anomaly-detection-internal:latest
```

Comment:
Using `latest` tag is not recommended. I see there is another tag in the registry. Can we change to that?

Reply:
Most of the images we import use `latest` tag. And others specify no tag, which also defaults to latest. See for yourself. It allows for a scheduled import.

https://github.com/AICoE/idh-manifests/blob/production/jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml

I don't see a reason in locking this to a version on a live project, if an image stream update PR takes 2 months to be merged.

Reply:
Lock to a specific tag other than latest will avoid break the image/notebook/whatever you are deploying. :)

If you are fine with this, I will be fine as well.

Reply:
> Lock to a specific tag other than latest will avoid break the image/notebook/whatever you are deploying

I don't see a way it can break even when the latest is applied. You spin up a pod with a certain image sha. That stays for the pod lifetime. Also there's a mapping between the source `DockerImage` tag to the `ImageStream` which should be respected. The `ImageStream` tag is named `latest` as well and is scheduled to be imported (every 15 mins by default). If the underlying image changes for example during pod scheduling, the pod is still scheduled with the old image sha since it doesn't get cleaned up by garbage collector because the image is being used.